hie-bios.cabal

```diff
@@ -66,7 +66,7 @@ Extra-Source-Files:     ChangeLog
                         tests/projects/simple-stack/simple-stack.cabal
 
 
-tested-with: GHC==8.6.4, GHC==8.4.4, GHC==8.2.2
+tested-with: GHC==8.6.4, GHC==8.4.4, GHC==8.2.2, GHC==8.8.1
```

Comment:
Strong statement to write tested with when we don't. Again, 8.6.5 would be better

Reply:
This was used to generate the .travis.yaml config. Modified it to reflect the actual tested ghc versions.